### PR TITLE
Release notes changes for 1.25+ck2

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

**Comment by addyess** [comment]
@charmed-kubernetes/canonical-k8s-team the inclusive naming checks fail due to links to upstream

File: pages/k8s/1.25/release-notes.md [hunk 1]
```diff
@@ -13,6 +13,41 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
+# 1.25+ck2 Bugfix release 
+
+### September ??, 2022 - `charmed-kubernetes --channel 1.25/stable`
```
Comment (kwmonroe) [inline pages/k8s/1.25/release-notes.md:18]:
```suggestion
### September 30, 2022 - `charmed-kubernetes --channel 1.25/stable`
```

File: pages/k8s/1.25/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,41 @@ layout: [base, ubuntu-com]
...
+
+- IPv6DualStack in Kubernetes-Control-Plane / Kubernetes-Worker [LP#1990455](https://bugs.launchpad.net/bugs/1990455)
+
+  The feature gate [`IPv6DualStack=true`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#other-cleanup-or-flake)
+  is the default since `1.21`, and GA since `1.23`, so this flag was removed in `1.25` in the upstream source.
+  As this charm release supports 1.22 through 1.25, there's no need for this flag anymore and it is being removed.
+
+- GCP snap in GCP-Integrator [LP#1988865](https://bugs.launchpad.net/bugs/1988865)
+
+  The snap installed by the gcp-integrator charm to manage gcp resources collided with the sanp
```
Comment (kwmonroe) [inline pages/k8s/1.25/release-notes.md:40]:
```suggestion
  The snap installed by the gcp-integrator charm to manage gcp resources collided with the snap
```

**Review by kwmonroe — APPROVED** [review]
Couple minor spelling updates; otherwise lgtm.

**Review by evilnick — APPROVED** [review]

**Pull request merged**